src/dialects/mysql/schema/tablecompiler.js

```diff
@@ -194,14 +194,13 @@ assign(TableCompiler_MySQL.prototype, {
       })
     );
   },
-  index(columns, indexName) {
+  index(columns, indexName, indexType) {
```

Comment:
can you also add test for this? probably just a unit test for query generation would suffice

Reply:
and there probably are existing tests for pg already

Reply:
The format of the tests is confusing to me. Many tests are failing and I do not know why. https://travis-ci.org/tgriesser/knex/jobs/450867352 (The tests fail even before the change)

I'll look at the postgre tests to see if I can understand.

Reply:
Done.

Reply:
```
    it('test adding index', function() {
      tableSql = client
        .schemaBuilder()
        .table('users', function() {
          this.index(['foo', 'bar'], 'baz');
        })
        .toSQL();

      equal(1, tableSql.length);
      expect(tableSql[0].sql).to.equal(
        'alter table `users` add index `baz`(`foo`, `bar`)'
      );
    });
```

This is an example of a test failing. I assume structure of an index statement changed even when index type is not specified, which is actually probably not something that we want - maybe excessive space somewhere?..

Reply:
@douglasjunior Feel free to ignore Oracle tests failing, they've been like that for a while already. But MySQL ones should be passing.

Reply:
Now I think I get it, please, could you check it again?